src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -119,7 +118,12 @@ export function textFormatter(row: number | undefined, cell: any | undefined, va
 
 export function iconCssFormatter(row: number | undefined, cell: any | undefined, value: any, columnDef: any | undefined, dataContext: any | undefined): string {
 	if (isCssIconCellValue(value)) {
-		return `<div role="image" title="${escape(value.title)}" aria-label="${escape(value.title)}" class="grid-cell-value-container icon codicon slick-icon-cell-content ${value.iconCssClass}"></div>`;
+		if ((<any>value).text) {
```

Comment:
I took another look at this, so you want this so that you can have an icon next to the text? in that case you should reuse a plugin we already have: https://github.com/microsoft/azuredatastudio/blob/main/src/sql/workbench/contrib/dashboard/browser/widgets/explorer/explorerTable.ts#L200

![image](https://user-images.githubusercontent.com/13777222/171757713-6e1ccd64-72d6-4299-a943-895d5aaf7cb0.png)

Reply:
now that you are using the textWithIconColumn plugin, do you still need the changes here?